src/Entity/IdeasWorkshop/Theme.php

```diff
@@ -86,11 +86,23 @@ class Theme implements EnabledInterface
      */
     protected $imageName;
 
-    public function __construct(string $name = '', string $imageName = null, bool $enabled = false)
+    /**
+     * @Assert\GreaterThanOrEqual(1)
+     *
+     * @Gedmo\SortablePosition
+     *
+     * @ORM\Column(type="smallint", options={"unsigned": true})
```

Comment:
Can you declare this annotation first ? ( We declare `@ORM` annotations always first )

Reply:
hue